sdcm/utils/issues.py

```diff
@@ -0,0 +1,93 @@
+import re
+import sys
+import logging
+from functools import lru_cache
+from dataclasses import dataclass
+
+import github
+import github.Auth
+from github.GithubException import UnknownObjectException
+
+from sdcm.keystore import KeyStore
+from sdcm.sct_config import SCTConfiguration
+from sdcm.sct_events.base import Severity
+from sdcm.sct_events.system import TestFrameworkEvent
+
+scylla_issue_pattern = re.compile(
+    r"^\s*((((?P<user_id>[\w:-]+)/)?(?P<repo_id>[\w:-]+))?#)?(?P<issue_id>\d+)\s*$", re.IGNORECASE)
+
+scylla_issue_https_pattern = re.compile(
```

Comment:
Same here.

`github_issue_https_pattern` ?

Reply:
changed the name